src/Parser/Warrior/Fury/Modules/Features/Abilities.js

```diff
@@ -108,47 +116,53 @@ class Abilities extends CoreAbilities {
       noCanBeImproved: true,
       extraSuggestion: 'Only used in a combat with AoE or as a filler with the Wrecking Ball talent',
     },
+    {
+      spell: SPELLS.BERSERKER_RAGE,
+      category: Abilities.SPELL_CATEGORIES.UTILITY,
+      getCooldown: haste => 45,
+      recommendedEfficiency: 0.95,
```

Comment:
Is using Berserker Rage on cooldown this critical? Keep in mind that a recommendedEfficiency of 0.95 for a spell with 45 second CD means that you'd consider it an issue for the player to wait longer than 2 seconds to cast it after it becomes available. Basically you're asserting there's never a reason to hold this.

I don't know much about Fury Warriors, so this may be correct, just double checking.

Reply:
You are probably right that it is unrealisticly harsh. I'll tune it down a bit, but I don't see it being of interest any time soon, as there are currently 0 logged kills on warcraftlogs where anyone has used this talent.

Reply:
Doesn't matter if it's used a lot or not, if you make it, you should make it properly. 
What I sometimes use for unconventional abilities like this one, is deduct a bit of usage from a simc baseline profile with it, and work from there.

Reply:
That's a good way of determining realistic use - I'll do that. Thanks